blueprint/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: AWS EventBridge - Create a PagerDuty incident in response to OAuth Client deletes
+author: ronan.watkins
+indextype: blueprint
+icon: blueprint
+image: images/arch-eventbridge-lambda.png
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint demonstrates a Lambda function that creates a PagerDuty incident in response to OAuth Client deletes. This Blueprint includes a SAM template for the CloudFormation stack used in this Blueprint with Typescript and Python Lambda functions.
+---
+
+This Genesys Cloud Developer Blueprint demonstrates a Lambda function that creates a PagerDuty incident in response to OAuth Client deletes. This Blueprint includes a SAM template for the CloudFormation stack used in this Blueprint with Typescript and Python Lambda functions.
+
+![Diagram for the AWS EventBridge - Create a PagerDuty incident in response to OAuth Client deletes Blueprint](images/arch-eventbridge-lambda.png "Diagram for the AWS EventBridge - Create a PagerDuty incident in response to OAuth Client deletes Blueprint")
+
+* [Solution components](#solution-components "Goes to the Solutions components section")
+* [Software development kits (SDKs)](#software-development-kits-sdks "Goes to the Software development kits (SDKs) section")
+* [Prerequisites](#prerequisites "Goes to the Prerequisites section")
+* [Implementation steps](#implementation-steps "Goes to the Implementation steps section")
+* [Additional resources](#additional-resources "Goes to the Additional resources section")
+
+## Solution components
+
+* **PagerDuty** - PagerDuty is an incident management platform that provides reliable notifications, automatic escalations, on-call scheduling, and other functionality to help teams detect and fix infrastructure problems quickly.
+* **AWS SAM CLI** - A cross-platform CLI that provides a Lambda-like execution environment that lets you locally build, test, and debug applications defined by SAM templates.
+* **AWS Lambda** - A compute service that lets you run code without provisioning or managing servers.
+* **AWS EventBridge** - A serverless event bus that makes it easier to build event-driven applications at scale using events generated from your applications, integrated Software-as-a-Service (SaaS) applications, and AWS services.
+* **AWS CloudFormation** - A service that gives developers and businesses an easy way to create a collection of related AWS and third-party resources, and provision and manage them in an orderly and predictable fashion.
+* **Node.js** - An open-source, cross-platform JavaScript runtime environment.
+* **Python** - An interpreted high-level general-purpose programming language.
+
+## Software development kits (SDKs)
+
+* **PDJS** - A simple JavaScript wrapper for the PagerDuty APIs.
```

Comment:
Should link to some sort of source for these packages, like npm, pypi, github, etc. Same for the solution components above. It's not particularly difficult to look any of this stuff up, but linking helps give readers the confidence that they're finding the exact sources the author was referring to.